src/fhirtypes/ElementDefinition.ts

```diff
@@ -783,14 +913,20 @@ export class ElementDefinition {
     if (disabledFlags.length) {
       throw new DisableFlagError(disabledFlags);
     }
+
+    const connectedElements = this.findConnectedElements();
+
     if (isBoolean(mustSupport)) {
       this.mustSupport = mustSupport;
+      connectedElements.forEach(ce => (ce.mustSupport = mustSupport || ce.mustSupport));
     }
     if (isBoolean(summary)) {
       this.isSummary = summary;
+      connectedElements.forEach(ce => (ce.isSummary = summary));
```

Comment:
Yeah, this is sort of a magical implementation detail that feels less necessary every time it comes up, since FSH has no way of explicitly trying to set a flag to false. Basically, MS and ?! can't be turned off once they're on, so I'll add a comment to that effect.